### ISSUE TITLE
Coursera - Machine Learning(1)

### ISSUE THREAD

**Issue by BanSugyeong** [BanSugyeong/Machine-Learning-Study#1]
### Supervised Machine Learning : Regression and Classification

**Comment by BanSugyeong** [comment]
### Contents
- Machine Learning
- Learning Algorithm types
  - Supervised Learning
  - Unsupervised Learning
- Linear Regression Model
  - Simple Linear Regression
- Cost Function
  - minimze cost function with parameters
- Gradient descent Algorithm
  - Learning rate
  - Convex Function
  - Non-Convex Function

[240112_MachineLearning_BanSugyeong.pdf](https://github.com/BanSugyeong/Machine-Learning-Study/files/14318468/240112_MachineLearning_BanSugyeong.pdf)

**Comment by BanSugyeong** [comment]
### Contents
- Feedback
  - Loss function
  - MSE
  - RMSE
- Multiple Linear Regression
- Vectorization
- Gradient descent for MLR
- Classification
- Logistic Regression
- Regularization

[240119_MachineLearning_BanSugyeong.pdf](https://github.com/BanSugyeong/Machine-Learning-Study/files/14326623/240119_MachineLearning_BanSugyeong.pdf)

**Comment by BanSugyeong** [comment]
### Contents
- Feedback
  - implementation of MSE
  - implementation of RMSE
  - overfitting solutions
  - L1, L2 regularization
- Neural Networks
  - Layers
  - Forward Propagation
- Perceptron
  - single-layer perceptron
  - multi-layer perceptron
- Tensorflow
